std/testing/asserts.ts

```diff
@@ -21,15 +21,17 @@ export class AssertionError extends Error {
 
 export function _format(v: unknown): string {
   let string = globalThis.Deno
-    ? stripColor(Deno.inspect(v, {
+    ? Deno.inspect(v, {
```

Comment:
The intention of `stripColor()` here is to strip the colour that's added stylistically by `Deno.inspect()` based on types. This change will make regular objects look like this:
```
> deno eval 'console.log(Deno.inspect([123456789, 123456789, 123456789, 123456789, 123456789, 123456789, 123456789]).replaceAll("\x1b", "<esc>"))'
[
  <esc>[33m123456789<esc>[39m,
  <esc>[33m123456789<esc>[39m,
  <esc>[33m123456789<esc>[39m,
  <esc>[33m123456789<esc>[39m,
  <esc>[33m123456789<esc>[39m,
  <esc>[33m123456789<esc>[39m,
  <esc>[33m123456789<esc>[39m
]
```

Reply:
> his change will make regular objects look

Currently only sanitizing string inputs however.

Reply:
But actually; this should be handled at a lower level so I'll end up reverting that.